### DISCUSSION TITLE
Glasses connection

### DISCUSSION THREAD

**Discussion by JAlcaideH** [everysight-maverick/sdk#10] [General]
Hello, I have been developing an app. Now I want to connect it to the glasses itself.
First I am trying to connect the "Animations" example app to the glasses.
I used to connect the simulator this way:
- onCreate:
val options = hashSetOf("supportSimulator")
Evs.init(this).startExt(options)
Evs.instance().showUI("configure")

- initSDK:
if(hasConfiguredDevice()) connect()

This works with the simulator, so using the UI I can connect to it.

Then for the glasses connection I have:
- onCreate:
Evs.init(this).start()
Evs.instance().showUI("configure")

- initSDK:
if(hasConfiguredDevice()) connect()

But when I try to connect to the glasses after a few seconds I get the following error message in the App:
"App auth Failed ActivationError"

**Comment by benchuk** [comment]
Hi @JAlcaideH
it seems like you're on the right track technically. Please reach out to our support team at support@everysight.com for further assistance.
